### PR TITLE
BaSD-MS-162: add home page

### PR DESCRIPTION
**Pull request by fjmorabito**
In this feature, Home Page is added to the app.

You can fin:
- Sign up screen in the home page route.
- Open positions screen.
- Position detailed screen.
- Template to edit contact and log in screen

### CODE REVIEW

File: src/Components/MainPage/MainPage/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (veronabaudracco) [inline src/Components/MainPage/MainPage/index.js:1]: I believe this page should not exist. The Home page should be the list of open positions.

File: src/Components/MainPage/PositionDetailed/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+  }, []);
+
+  useEffect(() => {
+    return () => {
+      dispatch(cleanSelectedItem());
+    };
+  }, []);
+
+  return (
```
Comment (veronabaudracco) [inline src/Components/MainPage/PositionDetailed/index.js:28]: Every field should have a title: "Position", "Client", "Opened at"

File: src/Components/MainPage/PositionDetailed/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <div>
+      <div>
+        <h3>{selectedItem.professionalProfile?.name}</h3>
+        <p>{selectedItem.client?.name}</p>
+        <p>{selectedItem.createdAt}</p>
```
Comment (veronabaudracco) [inline src/Components/MainPage/PositionDetailed/index.js]: This field should only show the date, it doesn't matter what time the position was opened

**Review by veronabaudracco — CHANGES_REQUESTED** [review]

File: src/Components/MainPage/MainPage/index.js (hunk 1 above)
Comment (fjmorabito) [reply src/Components/MainPage/MainPage/index.js:1]: Well it depends on the group decision. It's not so bad to have a welcome page

File: src/Components/MainPage/PositionDetailed/index.js (hunk 2 above)
Comment (fjmorabito) [reply src/Components/MainPage/PositionDetailed/index.js:28]: Check Computrabajo or any other website. Its quite rare to have a title. Perhaps a description should be added

**Review by veronabaudracco — APPROVED** [review]

**Review by AgustinCarthery — APPROVED** [review]

**Pull request merged**